viewer/packages/utilities/src/events/PointerEventsTarget.ts

```diff
@@ -164,9 +163,8 @@ export class PointerEventsTarget {
     if (clickDuration >= MAX_CLICK_DURATION) {
       return false;
     }
-    const { offsetX, offsetY } = clickOrTouchEventOffset(event, this._domElement);
-    const distance = Math.sqrt(offsetX - this._downPosition.x) ** 2 + (offsetY - this._downPosition.y) ** 2;
```

Comment:
Haha, not sure what we ended up computing here :laughing:

Reply:
The distance to when last mouse down.